components/eventing-controller/controllers/backend/reconciler.go

```diff
@@ -622,7 +636,16 @@ func getSecretStringData(clientID, clientSecret, tokenEndpoint, grantType, publi
 }
 
 func (r *Reconciler) CreateOrUpdatePublisherProxy(ctx context.Context, backend eventingv1alpha1.BackendType) (*appsv1.Deployment, error) {
+	return r.CreateOrUpdatePublisherProxyDeployment(ctx, backend, true)
+}
+
+func (r *Reconciler) CreateOrUpdatePublisherProxyDeployment(
+	ctx context.Context,
+	backend eventingv1alpha1.BackendType,
+	withOwnerReference bool) (*appsv1.Deployment, error) {
```

Comment:
```suggestion
	withOwnerReference bool) (*appsv1.Deployment, error) {
```
let's give this a more 'booly' name; we use the 'with' prefix usually for options like [here](https://github.com/kyma-project/nats-manager/blob/9241d3b926bd16da195988aeeeac54ec9d8e23d5/testutils/options.go#L53)

Reply:
the flag is to say whether set owner reference or not. `hasOwnerReference` would mean whether it has owner reference. I can rename it to `setOwnerReference` if you this to the existing one?

Reply:
that sounds like a satisfactory solution.